tests/config.go

```diff
@@ -39,6 +46,15 @@ func NewConfig() *Config {
 	flag.StringVar(&cfg.TidbVersions, "tidb-versions", "v2.1.3,v2.1.4", "tidb versions")
 	flag.StringVar(&cfg.OperatorTag, "operator-tag", "master", "operator tag used to choose charts")
 	flag.StringVar(&cfg.OperatorImage, "operator-image", "pingcap/tidb-operator:latest", "operator image")
+	flag.StringVar(&cfg.GitRepoDir, "git-repo-dir", "/tidb-operator", "local directory to which tidb-operator cloned")
+	if home := homeDir(); home != "" {
+		flag.StringVar(&cfg.KubeConfigPath, "kubeconfig", filepath.Join(home, ".kube", "config"), "absolute path to the kubeconfig file")
+	} else {
+		// cannot get homedir, kube config file path must be provided explicitly
+		flag.StringVar(&cfg.KubeConfigPath, "kubeconfig", "", "absolute path to the kubeconfig file")
+	}
+	flag.BoolVar(&cfg.OutOfCluster, "out-of-cluster", false, "whether stability test runs out of cluster.")
```

Comment:
Maybe we can omit `out-of-cluster` flag,  an idiomatic way to load kube config is

- load config fro kubeconfig file if given
- load in-cluster config

Reply:
How to omit kubeconfig option (default to `~/.kube/config`) in this case?

Reply:
How about using `controller-runtime/pkg/client/config/config.go#GetConfig()`, seems that method is well designed.

Reply:
https://godoc.org/sigs.k8s.io/controller-runtime/pkg/client/config#GetConfig seems very good. It helps to make our CLI behave as other k8s CLI commands.